ox-reveal.el

```diff
@@ -362,6 +371,21 @@ content."
   :group 'org-export-reveal
   :type 'string)
 
+(defcustom org-reveal-klipsify-src nil
+  "Set to non-nil if you would like to make source code blocks editable in exported presentation."
+  :group 'org-export-reveal
+  :type 'boolean)
+
+(defcustom org-reveal-klipse-css "https://storage.googleapis.com/app.klipse.tech/css/codemirror.css"
+  "Location of the codemirror css file for use with klipse."
+  :group 'org-export-reveal
+  :type 'string)
+
+(defcustom org-reveal-klipse-js "https://storage.googleapis.com/app.klipse.tech/plugin_prod/js/klipse_plugin.min.js"
```

Comment:
there is a small catch here: for clojure src block evaluation we need another js file instead of this one: https://storage.googleapis.com/app.klipse.tech/plugin/js/klipse_plugin.js
This is a non-minized version of the klipse plugin that supports all the languages **including clojure**.
while the minimized version of the klipse plugin supports all the languages **excepted clojure**.

I see two possible solutions:

- not to include clojure in the list of languages that clojure supports with reveal
- provides `org-reveal-klipsify-src-full` that will load `https://storage.googleapis.com/app.klipse.tech/plugin/js/klipse_plugin.js`

I prefer the second one